core/deployment/src/main/java/io/quarkus/deployment/steps/MainClassBuildStep.java

```diff
@@ -213,25 +178,60 @@ MainClassBuildItem build(List<StaticBytecodeRecorderBuildItem> staticInitTasks,
 
         tryBlock = mv.tryBlock();
 
-        // Load the run time configuration
-        tryBlock.invokeStaticMethod(RunTimeConfigurationGenerator.C_CREATE_RUN_TIME_CONFIG);
+        //        // Load the bootstrap configuration
```

Comment:
This should all go away.  Might be worth tagging the commit for reference first.

Reply:
Oh darn! Sorry for missing that! Basically the PR has 2 commits the first approach and the new approach so everything should be in the history, so I think we can safely delete all this :P